examples/references/classification/imagenet/code/utils/exp_tracking.py

```diff
@@ -10,7 +10,7 @@
 from ignite.contrib.engines import common
 
 try:
-    import polyaxon_client.tracking
+    import polyaxon_client.tracking  # noqa: F401
```

Comment:
As we do not modify our own setup.cfg, technically, there is no need to add `# noqa: F401`, right ?

Reply:
But, F401 will be removed after we clean up all unused imports, isn't it?

Reply:
I was considering that previously but it looks like we will have to add a lot of those `# noqa: F401` inside `__init__.py`, right ?

Reply:
Yes unless we add them in `__all__`

Reply:
Sounds good !